docs/articles/reference/upgrading-to-v16.md

```diff
@@ -0,0 +1,258 @@
+## Upgrading to v0.16
+
+Bytewax v0.16 introduces major changes to the way that custom Bytewax inputs are created, as well as improvements to and windowing and execution. In this article, we'll cover some of the updates we've made to our API to make upgrading to v0.16 easier.
+
+## Input changes
```

Comment:
Should we also call out that `bytewax.inputs.KafkaInputConfig` has been moved to `bytewax.connectors.kafka.KafkaInput`?